Example/Example/ViewController.swift

```diff
@@ -6,20 +6,20 @@
 //  Copyright © 2019 Rover Labs Inc. All rights reserved.
 //
 
-import RoverCampaigns
+import RoverFoundation
 import UIKit
 
 class ViewController: UITabBarController {
     override func viewDidLoad() {
         super.viewDidLoad()
         
         // Add the notification center to the tab bar
-        let notificationCenter = RoverCampaigns.shared!.resolve(UIViewController.self, name: "notificationCenter")!
+        let notificationCenter = RoverFoundation.shared!.resolve(UIViewController.self, name: "notificationCenter")!
         notificationCenter.tabBarItem = UITabBarItem(title: "Notifications", image: nil, tag: 0)
         
         // Add the settings view controller to the tab bar.  Note that you typically wouldn't want to add this to the tab bar in your application.  Refer to the documentation.
-        let settingsViewController = RoverCampaigns.shared!.resolve(UIViewController.self, name: "settings")!
-        settingsViewController.tabBarItem = UITabBarItem(title: "Settings", image: nil, tag: 0)
+        let settingsViewController = RoverFoundation.shared!.resolve(UIViewController.self, name: "settings")!
+//        settingsViewController.tabBarItem = UITabBarItem(title: "Settings", image: nil, tag: 0)
```

Comment:
Did you intentionally leave this commented out?

Reply:
Not sure how that happened; I must have fat-fingered something. nice catch, thx